### PR TITLE
Template Repo

### PR DESCRIPTION
**Pull request by nabobalis**
Todo:
- [x] Get a review
- [x] Add actions CI
- [x] more to come when I recall what it was

For a follow up?
- [ ] set up RTD hook
- [ ] codecov?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Helioviewer Python API Wrapper
+------------------------------
+
+``hvpy`` is a Python API wrapper around the formal Helioviewer API.
```
Comment (dgarciabriseno) [inline README.rst:4]:
```suggestion
``hvpy`` is a Python API wrapper around the formal `Helioviewer API <https://api.helioviewer.org/docs/v2/>`_.
```
Can we add a link to the API?

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'hvpy'
+copyright = '2022, '
+author = ''
```
Comment (dgarciabriseno) [inline docs/conf.py:14]:
```suggestion
author = 'The Helioviewer Project'
```

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,111 @@
+[metadata]
+name = hvpy
+author = Helioviewer Project
+author_email = add@email.com
```
Comment (dgarciabriseno) [inline setup.cfg:4]: @wafels 
Should we still use contact@helioviewer.org

File: README.rst (hunk 1 above)
Comment (nabobalis) [reply README.rst:4]: Will do!

Comment (nabobalis) [reply README.rst]: Fixed

File: docs/conf.py (hunk 2 above)
Comment (nabobalis) [reply docs/conf.py]: Fixed

File: setup.cfg (hunk 3 above)
Comment (nabobalis) [reply setup.cfg]: I added this for the time being.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+v0.0.1 (2022-11-11)
```
Comment (nabobalis) [inline CHANGELOG.rst:1]: I decided that just manually editing a changelog file is probably all we need instead of any more complex system.

**Comment by nabobalis** [comment]
For me to check the CI is working as expected, we need to merge this in first and then see if actions is happy.

File: CHANGELOG.rst (hunk 4 above)
Comment (dgarciabriseno) [reply CHANGELOG.rst:1]: That's alright

File: hvpy/api.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+def fake_api_call():
+    """
+    _summary_
+
+    Returns
+    -------
+    _type_
+        _description_
+    """
```
Comment (dgarciabriseno) [inline hvpy/api.py:12]: Do these comments get parsed into the docs?

**Review by dgarciabriseno — APPROVED** [review]

File: hvpy/api.py (hunk 5 above)
Comment (nabobalis) [reply hvpy/api.py:12]: They will be, I just created this to ensure the doc build works.

**Pull request merged**